### DISCUSSION TITLE
Distributed tracing

### DISCUSSION THREAD

**Discussion by evdokimovn** [openfga/community#322] [General]
There is a support for tracing but as far as I can tell if a request contains trace propagation headers those are not extracted from the original request. To me it seems that it should happen in the PB code (https://github.com/openfga/api/blob/main/proto/openfga/v1/openfga_service.pb.gw.go). Or is not entirely true? Did anyone had any luck with distributed tracing?

**Comment by ewanharris** [comment]
👋🏻 Hey @evdokimovn,

Which version of OpenFGA are you using? We shipped a [fix for grpc trace propagation](https://github.com/openfga/openfga/pull/1624) as part of v1.5.5 that I think might potentially solve the issue.

**Reply by evdokimovn** [reply]
Hey!

Yes I was on an older version. Worked like a charm after and upgrade ⭐

**Comment by aaguiarz** [comment]
@evdokimovn can you elaborate on how you are using tracing and what problem were you trying to solve? Thanks!
